### PR TITLE
Feat/gallery image titles

### PR DESCRIPTION
**Pull request by talevy17**

### CODE REVIEW

File: packages/plugin-gallery/web/src/components/gallery-controls/gallery-image-settings.js [hunk 1]
```diff
@@ -192,7 +192,7 @@ class ImageSettings extends Component {
                   </button>
                 </div>
                 <SettingsSection
-                  ariaProps={{ 'aria-label': 'image properties', role: 'region' }}
+                  ariaProps={{ title: 'image properties', role: 'region' }}
```
Comment (bchmn) [inline packages/plugin-gallery/web/src/components/gallery-controls/gallery-image-settings.js]: why was this changed? it should remain `aria-label`

File: packages/plugin-gallery/web/src/gallery-viewer.jsx [hunk 2]
```diff
@@ -84,32 +89,56 @@ class GalleryViewer extends React.Component {
       // container size change callback
       case 'GALLERY_CHANGE':
         // ignore thumbnails layout
-        if (this.state.styleParams.galleryLayout === 3) {
-          return;
-        }
+        // if (this.state.styleParams.galleryLayout === 3) {
+        //   return;
+        // }
```
Comment (bchmn) [inline packages/plugin-gallery/web/src/gallery-viewer.jsx]: why is this commented out?

File: packages/plugin-gallery/web/src/gallery-viewer.jsx [hunk 3]
```diff
@@ -58,9 +58,14 @@ class GalleryViewer extends React.Component {
   stateFromProps = props => {
     const defaults = getDefault();
     const items = props.componentData.items || defaults.items;
-    const styleParams = Object.assign(defaults.styles, props.componentData.styles || {});
-
+    let styleParams = Object.assign(defaults.styles, props.componentData.styles || {});
+    styleParams = this.hasTitle(items)
```
Comment (bchmn) [inline packages/plugin-gallery/web/src/gallery-viewer.jsx]: this is ugly, please create a `getStyleParams` method that handles all this logic

File: packages/plugin-gallery/web/src/gallery-viewer.jsx (hunk 2 above)
Comment (talevy17) [reply packages/plugin-gallery/web/src/gallery-viewer.jsx]: It's irrelevant with the gallery key
kept it commented for now because I have no idea when will the pro gallery bugs be fixed and to what extent
